validator/remote/src/main/java/tech/pegasys/teku/validator/remote/sentry/SentryNodesConfigLoader.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.remote.sentry;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.google.common.annotations.VisibleForTesting;
+import java.io.FileNotFoundException;
+import java.nio.charset.StandardCharsets;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes;
+import tech.pegasys.teku.infrastructure.http.UrlSanitizer;
+import tech.pegasys.teku.infrastructure.io.resource.ResourceLoader;
+import tech.pegasys.teku.provider.JsonProvider;
+
+public class SentryNodesConfigLoader {
+
+  private static final Logger LOG = LogManager.getLogger();
+  private final JsonProvider jsonProvider;
+  private final ResourceLoader resourceLoader;
+
+  public SentryNodesConfigLoader() {
+    this(new JsonProvider(), ResourceLoader.urlOrFile("application/json"));
```

Comment:
we don't want to be creating a new JsonProvider, that will go away though if we use deserializableType, so i think we're likely going to do that...